modules/installation-vsphere-installer-infra-requirements0.adoc

```diff
@@ -0,0 +1,299 @@
+
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-networking_{context}"]
+== Networking requirements
+
+You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines.
+ifdef::restricted[]
+The VM in your restricted network must have access to vCenter so that it can provision and manage nodes, persistent volume claims (PVCs), and other resources.
+endif::restricted[]
+Additionally, you must create the following networking resources before you install the {product-title} cluster:
+
+[NOTE]
+====
+It is recommended that each {product-title} node in the cluster must have access to a Network Time Protocol (NTP) server that is discoverable via DHCP. Installation is possible without an NTP server. However, asynchronous server clocks will cause errors, which NTP server prevents.
+====
+
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-_{context}"]
+=== Required IP Addresses
+An installer-provisioned vSphere installation requires two static IP addresses:
+
+* The **API** address is used to access the cluster API.
+* The **Ingress** address is used for cluster ingress traffic.
+
+You must provide these IP addresses to the installation program when you install the {product-title} cluster.
+
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-dns-records_{context}"]
+=== DNS records
+You must create DNS records for two static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+
+.Required DNS records
+[cols="1a,5a,3a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+|API VIP
+|`api.<cluster_name>.<base_domain>.`
+|This DNS A/AAAA or CNAME record must point to the load balancer
+for the control plane machines. This record must be resolvable by both clients
+external to the cluster and from all the nodes within the cluster.
+
+|Ingress VIP
+|`*.apps.<cluster_name>.<base_domain>.`
+|A wildcard DNS A/AAAA or CNAME record that points to the load balancer that targets the
+machines that run the Ingress router pods, which are the worker nodes by
+default. This record must be resolvable by both clients external to the cluster
+and from all the nodes within the cluster.
+|===
+
+ifeval::["{context}" == "installing-restricted-networks-installer-provisioned-vsphere"]
+:!restricted:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-vmc"]
+:!restricted:
+endif::[]
```

Comment:
what's the purpose of this update? Seems unrelated with this PR.

Reply:
Unwanted file removed.